### DISCUSSION TITLE
Is there an "Asset Materialization Failure Sensor"?

### DISCUSSION THREAD

**Discussion by achen187** [dagster-io/dagster#17572] [Q&A] [answered]
Similar to the run_failure_sensor for jobs, I'm wondering if there's an "materialization_failure_sensor" where you specify a list of asset_keys, and the sensor activates on failure. 

Something analogous to the link below, but for assets instead of jobs:

https://docs.dagster.io/concepts/partitions-schedules-sensors/sensors#run-failure-sensor

**Comment by sryza** [comment, marked as answer]
Hi @achen187 - not currently. I'm going to turn this into an issue so we can track it as a feature request.

**Reply by sryza** [reply]
Here's the issue: https://github.com/dagster-io/dagster/issues/17573
